iris/tests/pipeline/chat/test_ask_user_pipeline_unit.py

```
@@ -0,0 +1,651 @@
+    rendered = system_message.prompt.template
+    assert VERDICT_DEPENDENT not in rendered
+    # Braces from the rendered verdict template must be doubled so LangChain
+    # does not try to interpret them as template variables.
+    assert "{{question}}" in rendered
+    assert "prefix" in rendered and "suffix" in rendered
+
+
+def test_assess_answer_reports_error_on_malformed_json():
+    pipeline, state, _msg, callback = _assess_answer_state("not json at all")
+
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)